homework_14_140123/src/MethodsImplementation.java

```diff
@@ -0,0 +1,409 @@
+//    считать его и вывести на екран кол-во символов в файле,
+//    кол-во слов в файле.(Два значения)
+
+    public static void countSymbolsAndFiles() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter file name: ");
+        String fileName = scanner.nextLine();
+        File file = new File(fileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(file, true));
+            bufferedWriter.write("First line\n");
+            bufferedWriter.write("Second line\n");
+            bufferedWriter.write("Third line\n");
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
+                bufferedWriter.close();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+        BufferedReader bufferedReader = null;
+        String line = null;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(file));
+            int count = 0;
+            StringBuilder stringBuilder = new StringBuilder();
+            line = null;
+            while (true) {
+                int i;
+                if ((i = bufferedReader.read()) == -1) {
+                    break;
+                } else {
+                    line = String.valueOf(stringBuilder.append((char) i));
+                    count++;
+                }
+            }
+            System.out.println("Characters: " + count);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
+                bufferedReader.close();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+        String newLine = line.replace("\n", " ");
+        String[] arrString = newLine.split(" ");
+        int words = 0;
+        for (String element : arrString) {
+            words++;
+        }
+        System.out.println("Words: " + words);
+    }
+
+
+    //Task4
+//    Создать текстовый файл произвольного содержания и переименовать его
+    public static void renameFile() {
+        File file = new File("TestFile.txt");
+        try {
+            file.createNewFile();
+            System.out.println(file + " is created");
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(file));
+            bufferedWriter.write("First line\n");
+            bufferedWriter.write("Second line\n");
+            bufferedWriter.write("Third line\n");
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
```

Comment:
you don't need try block here, just close stream

Reply:
fixed. using try with resources.